activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -18,6 +20,16 @@ module FinderMethods
     #   Person.find([1])        # returns an array for the object with ID = 1
     #   Person.where("administrator = 1").order("created_on DESC").find(1)
     #
+    # ==== Find a composite primary key model
+    #   TravelRoute.primary_key = [:origin, :destination]
+    #   TravelRoute.find("Ottawa", "London") # returns the object for the route from Ottawa to London
+    #   TravelRoute.find(["Paris", "Montreal"]) # returns an array the object for the route from Paris to Montreal
```

Comment:
Won't this return a single `TravelRoute` record?

Reply:
Yep, it was supposed to be double wrapped like `[[ID]]` and the one above was supposed to represent `.find(ID)` 
Should be fixed now!